src/Core/System/SystemConfig/SystemConfigService.php

```diff
@@ -88,10 +93,16 @@ public function get(string $key, ?string $salesChannelId = null)
                 continue;
             }
 
-            return null;
+            $event = new SystemConfigLoadedEvent($key, null, $salesChannelId);
```

Comment:
not 100% sure if we need it for the null values (missing values), for now I added it but I can remove those again.